secio/src/codec/hmac_compat/mod.rs

```diff
@@ -0,0 +1,45 @@
+#[cfg(not(target_os = "unknown"))]
```

Comment:
should be `target_arch` ?
```suggestion
#[cfg(not(target_arch = "wasm32"))]
```

Reply:
```
x86_64-pc-windows-gnu
x86_64-pc-windows-msvc
wasm32-unknown-emscripten
wasm32-unknown-unknown
wasm32-wasi
x86_64-unknown-linux-gnu
```

Obviously the third one is os, I think there is no problem writing this way